core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -191,9 +191,9 @@
      */
     public static final class DETAILS {
 
-        private static final String DETAILS_PREFIX = "details.";
```

Comment:
Ok, I think we should keep the variable 'DETAILS_PREFIX' as we do similar for the others.

The actual string `path_details` vs. `details` is harder to decide as path_details is longer but would be align with the Java class. I'm still preferring just `details` as this is directly attached to a path and also it is not clear if the class PathDetails remains or if it is replaced by `List<Details>` and `PathDetailsBuilder`. But the most important thing is: if we name the parameter `x` then the json entry should be also `x`